modules/diskq/qdisk.c

```diff
@@ -1432,6 +1432,13 @@ _load_qdisk_file(QDisk *self, const gchar *filename, GQueue *qout, GQueue *qback
   return FALSE;
 }
 
+static gboolean
+_set_qdisk_file_defaults(QDisk *self)
```

Comment:
I think this is not a good name.
Since I can't suggest a better one, I believe `_create_header()` should be kept in `qdisk_start()`.
Creating the header is a huge side effect of this function, I think this function should not exist.

Reply:
As agreed IRL, renamed it to `_init_qdisk_file()`.